### PR TITLE
BaSP-MR-104: Implements table components

### PR DESCRIPTION
**Pull request by YhanahiRosenthal**
Table component implemented

### CODE REVIEW

File: src/Components/Subscriptions/index.js [hunk 1]
```diff
@@ -1,11 +1,71 @@
...
       <h2>Subscriptions</h2>
-      <SubscriptionsTable />
+      <AddButton entity="Suscription" createMode={createMode} />
+      <TableComponent
+        columnTitleArray={columnTitleArray}
+        data={subscription}
+        handleClick={handleClick}
+        deleteButton={deleteSubscription}
+        valueField={valueField}
+        // arrayAndObject={arrayAndObject}
```
Comment (FrancoLelli) [inline src/Components/Subscriptions/index.js:63]:
```suggestion

```

**Review by FrancoLelli — CHANGES_REQUESTED** [review]
Delete comment!

**Review by CristianLotorto — APPROVED** [review]
fix the commented line, the rest is ok!

**Comment by maurojjzz** [comment]
[SUBSCRIPTION - Table](https://trello.com/c/iayq47wG/104-subscription-table)

**Pull request merged**